python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -380,6 +378,30 @@ def default_group_fn(node_info: Mapping[str, Any]) -> Optional[str]:
     if dbt_group:
         return dbt_group
 
+    return None
+
+
+def default_group_fn_fallback_to_directory(node_info: Mapping[str, Any]) -> Optional[str]:
```

Comment:
need API string for args
```suggestion
def default_group_fn_fallback_to_directory(dbt_resource_props: Mapping[str, Any]) -> Optional[str]:
```